invertedai_cpp/invertedai/blame_response.h

```diff
@@ -0,0 +1,77 @@
+#ifndef BLAME_RESPONSE_H
+#define BLAME_RESPONSE_H
+
+#include "data_utils.h"
+#include "externals/json.hpp"
+
+#include <string>
+#include <vector>
+
+using json = nlohmann::json;
+
+namespace invertedai {
+
+class BlameResponse {
+private:
+  std::vector<int> agents_at_fault_;
+  std::optional<float> confidence_score_;
+  std::optional<std::map<int, std::vector<std::string>>> reasons_;
+  std::optional<std::vector<std::vector<unsigned char>>> birdviews_;
+  json body_json_;
+
+  void refresh_body_json_();
+
+public:
+  BlameResponse(const std::string &body_str);
+  /**
+   * Serialize all the fields into a string.
+   */
+  std::string body_str();
+
+  // getters
+  /**
+   * Get a tuple containing all agents predicted to be at fault. If empty, BLAME
+   * has predicated no agents are at fault.
```

Comment:
Update this documentation to lose references to python variable types (tuple etc.)